src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/CountFunctions.txt

```diff
@@ -27,3 +27,27 @@ Errors: Error 22-25: Expected boolean. We expect a boolean (true/false) at this
 
 >> CountIf([1, 2, 3, 4], "abc")
 Errors: Error 22-27: Expected boolean. We expect a boolean (true/false) at this point in the formula.
+
+>> CountA([1,2,3,Blank()])
```

Comment:
>>> [](http://example.com/codeflow?start=0&length=2)

nit - can you put these in their own .txt file? (We should have split up CountRows, CountIf into their own files too)

Reply:
Done.